pkg/util/print/print_test.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package print
+
+import (
+	"testing"
+)
+
+func ptrint(i int) *int {
+	return &i
+}
+
+func ptrstr(s string) *string {
+	return &s
+}
+
+func TestPrettyPrintObject(t *testing.T) {
```

Comment:
based on the previous suggestion, maybe we could write tests to assert `spew.Sprintf(format, a)` will be called with the right format.